dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -12,7 +12,7 @@
 import copy
 import abc
 import warnings
-import sys
+from itertools import chain, zip_longest
```

Comment:
I think we may want to just import itertools and use the functions accordingly, that way we know what namespace they're from. I can imagine a function `chain`

Reply:
fixed